aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/ConflictResolver.java

```diff
@@ -88,11 +99,28 @@
             });
     }
 
+    @Nullable
+    private <T extends Model> T getT(@NonNull PendingMutation<T> pendingMutation, T remote) {
```

Comment:
Is it possible to use more descriptive name than `getT` based on what `T` is? maybe `getMutatedItem`  or `getLocalMutatedItem` etc?

Reply:
Done.